src/openrct2/actions/RideCreateAction.cpp

```diff
@@ -308,6 +312,7 @@ GameActions::Result RideCreateAction::Execute() const
     ride->income_per_hour = kMoney64Undefined;
     ride->profit = kMoney64Undefined;
     ride->connected_message_throttle = 0;
+    ride->drops = 0;
```

Comment:
Can't we just `memset` the `ride` to zeros and only set non-zero values here? Seems much less error-prone

Reply:
Ride does not satisfy std::is_pod which means UB territory.